src/introduction.adoc

```diff
@@ -90,13 +90,28 @@ RISC-V.
 It also adds a mode bit in the encoding of capabilities to
 allow changing the current CHERI execution mode using indirect jump
 instructions.
```

Comment:
We should delete, or at least modify, this sentence which is old. We also support changing the mode with modesw which is more efficient than the indirect jumps

Reply:
I've removed that text